### PR TITLE
Couple QoL changes around loading a file

### PR DESCRIPTION
**Pull request by mriswithe**
Set the default filter to the hades *.sav files
Return without trying to load the file if the user cancels out of the file dialog
Use Pathlib to get the path to the My Documents/Saved Games/Hades folder that hades uses and make the load file dialog start there

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -78,10 +79,15 @@ def open_file_name_dialog(self):
         fileName, _ = QFileDialog.getOpenFileName(
             self,
             "QFileDialog.getOpenFileName()",
-            "",
+            str(Path('~/My Documents/Saved Games/Hades').expanduser()),
```
Comment (zsennenga) [inline main.py:82]: Nice, helpful.

**Review by zsennenga — APPROVED** [review]

**Comment by zsennenga** [comment]
LGTM, I'll do a new build this weekend.

**Comment by mriswithe** [comment]
I had done some hacking on the hades save file back in feb 2020, I had even gone so far as writing my own code to read and write luabins... before I realized someone had already written one. Might poke around with a few other pieces of the code in the future too if that is alright!

**Comment by zsennenga** [comment]
Yeah no worries, be my guest. Also, feel free to merge this whenever.

**Comment by mriswithe** [comment]
Oh I am newish to collaborating on GitHub, I will merge it tonight hah

**Comment by mriswithe** [comment]
I don't think that I have the ability to merge this since I don't have write access. I might be doing something dumb or wrong

**Comment by zsennenga** [comment]
Lol whoops. I don't manage many open source repos, I assumed my approval was enough to let you merge this.

**Comment by mriswithe** [comment]
Learnings all around !

**Pull request merged**